tests/unit_tests/pyanaconda_tests/modules/storage/partitioning/test_module_scheduler.py

```diff
@@ -169,7 +169,7 @@ def test_get_default_file_system(self):
     def test_get_supported_raid_levels(self):
         """Test GetSupportedRaidLevels."""
         assert self.interface.GetSupportedRaidLevels(DEVICE_TYPE_MD) == \
-            ['linear', 'raid0', 'raid1', 'raid10', 'raid4', 'raid5', 'raid6']
```

Comment:
Is this intended to be there ? I find it strange this is part of an infra PR ?

Reply:
Well it's not an infra PR.
It's a PR with some fixes that are needed to re-enable building centos10 and centos9 containers

Reply:
Even the 'infra' marked commit - has tests changes.